woocommerce/woocommerce
# Update coming soon pages to use cardo and inter fonts via editor

### Changes proposed in this Pull Request:

Follow-up to https://github.com/woocommerce/woocommerce/pull/47417:

1. Use site editor settings to set fonts in coming soon pages
2. Add defensive optional type to ensure remove risk of crashing
3. Clean up style CSS in entire site template



### How to test the changes in this Pull Request:

#### Test with TT24 theme

1. Go to `WooCommerce > Settings > Site visibility`
4. Ensure `Coming soon` is selected and `Restrict to store pages only` is disabled
5. Go to `Appearance > Editor > Templates > Page: Coming soon` and edit (ensure no customizations were in place)
6. Select the center heading and open up `Settings > Block > Typography`
7. Observe `Cardo` is pre-selected
8. Select the `Log out` button and open up `Settings > Block > Typography` 
6. Observe `Inter` is pre-selected
7. Select the site title component on top left of the screen and open up `Settings > Block > Typography` 
6. Observe `Inter` is pre-selected
1. Go to `WooCommerce > Settings > Site visibility`
3. Ensure `Coming soon` is selected and `Restrict to store pages only` is enabled
4. Go to `Appearance > Editor > Templates > Page: Coming soon` and edit (ensure no customizations were in place)
4. Select `Great things are on the horizon` and open up `Settings > Block > Typography` 
6. Observe `Cardo` is pre-selected
4. Select `Something big is brewing!` and open up `Settings > Block > Typography` 
6. Observe `Inter` is pre-selected

#### Test with non-TT24 theme
1. Install and activate the [FSE Empire](https://wordpress.org/themes/fse-empire/) theme
1. Test again with the steps in above instructions ⬆️ starting from step 2 

#### Optional: Test frontend
1. In an incognito window, open up the store's front end at `/cart`
3. Observe the coming soon pages are working as expected

#### Optional: Test non-FSE themes
1. Install and activate the [Storefront](https://wordpress.org/themes/storefront/) theme
1. In an incognito window, open up the store's front end at `/cart`
2. Observe the coming soon pages are working as expected

<img width="1086" alt="image" src="https://github.com/woocommerce/woocommerce/assets/3747241/c22956b1-7cfa-48ea-a404-3ebb236ce03b">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
